lib/sqlalchemy/ext/hybrid.py

```diff
@@ -764,7 +764,7 @@ def __call__(s, self: Any) -> _T_co:
 
 
 class _HybridSetterType(Protocol[_T_con]):
-    def __call__(self, instance: Any, value: _T_con) -> None:
+    def __call__(s, self: Any, value: _T_con) -> None:
```

Comment:
it's the parameter names that have to match...weird

Reply:
Indeed